types/chrome/test/index.ts

```diff
@@ -1916,3 +1916,56 @@ async function testOffscreenDocument() {
     await chrome.offscreen.hasDocument();
     await chrome.offscreen.closeDocument();
 }
+
+// https://developer.chrome.com/docs/extensions/reference/fileSystemProvider/
+function testFileSystemProvider() {
+    // Checking onGetMetadataRequested, its option and EntryMetadata.
+    chrome.fileSystemProvider.onGetMetadataRequested.addListener((
+        options: chrome.fileSystemProvider.MetadataRequestedEventOptions,
+        successCallback: (
+          metadata: chrome.fileSystemProvider.EntryMetadata
+        ) => void,
+        errorCallback: (error: string) => void,
+    ) => {
+        const entryMetadata: chrome.fileSystemProvider.EntryMetadata = {};
+        if (options.isDirectory) {
+          entryMetadata.isDirectory = true;
+        }
+        if (options.name) {
+          entryMetadata.name = 'some-file.txt';
+        }
+        if (options.size) {
+          entryMetadata.size = 42;
+        }
+        if (options.mimeType) {
+          entryMetadata.mimeType = 'text/plain';
+        }
+    }
+    );
+
+    // Checking onReadDirectoryRequested.
+    chrome.fileSystemProvider.onReadDirectoryRequested.addListener((
+        options: chrome.fileSystemProvider.DirectoryPathRequestedEventOptions,
+        successCallback: (
+          entries: chrome.fileSystemProvider.EntryMetadata[],
+          hasMore: boolean
+        ) => void,
+        errorCallback: (error: string) => void) => {
+        }
+      );
+
+    // Checking onGetActionsRequested.
+    chrome.fileSystemProvider.onGetActionsRequested.addListener((
+        options: chrome.fileSystemProvider.GetActionsRequestedOptions,
+        successCallback: (actions: chrome.fileSystemProvider.Action[]) => void,
+        errorCallback: (error: string) => void) => {}
+    );
+
+    // Checking onExecuteActionRequested.
+    chrome.fileSystemProvider.onExecuteActionRequested.addListener((
+        options: chrome.fileSystemProvider.ExecuteActionRequestedOptions,
+        successCallback: () => void,
+        errorCallback: (error: string) => void) => {}
+      );
+
+}
```

Comment:
```suggestion
      );
}
```

Reply:
I've re-formatted these new lines using prettier.

BTW: Both of these files are very off in the format comparing to prettier.

I can do a follow up PR reformatting them with prettier if this is desirable.

Reply:
Yeah the formatting is a bit off but I don't know how this repo deals with that